### PR TITLE
Backend: Get details on a series or movie

### PR DESCRIPTION
**Pull request by JakWai01**
Closes https://github.com/alphahorizonio/consumat.io/issues/33

### CODE REVIEW

File: consumatio/external/tmdb.py [hunk 1]
```diff
@@ -33,6 +35,11 @@ def get_movie_images(self, movie_id):
         data = self.get_data(query, self.db)
         return movie_images_to_dict(data)
 
+    def get_movie_credits(self, movie_id):
+        query = f'https://api.themoviedb.org/3/movie/{movie_id}/credits?api_key=c4bd02adee6e73c4d17e69b039267c90'
```
Comment (pojntfx) [inline consumatio/external/tmdb.py:39]: Please pull the latest changes from the `main` branch and adjust the `api_key` value accordingly.

**Review by pojntfx — CHANGES_REQUESTED** [review]

File: consumatio/usecases/episode_details.py [hunk 2]
```diff
@@ -12,7 +12,7 @@ def episode_details(tmdb, code, season_number, episode_number):
         "overview": dict_episode_details.get("overview"),
         "airDate": dict_episode_details.get("air_date"),
         "voteAverage": dict_episode_details.get("vote_average"),
-        "stills": dict_episode_images.get("stills"),
+        "still": dict_episode_images.get("stills"),
```
Comment (pojntfx) [inline consumatio/usecases/episode_details.py:15]: This applies to those below as well, are you sure it is `dict_episode_images.get("stills")` and not `dict_episode_images.get("still")`?

**Review by pojntfx — CHANGES_REQUESTED** [review]

File: consumatio/usecases/episode_details.py (hunk 2 above)
Comment (JakWai01) [reply consumatio/usecases/episode_details.py:15]: Yes! The internal representation was an array which got reduced to a single still. Thats why there is a difference in the naming.

Comment (pojntfx) [reply consumatio/usecases/episode_details.py:15]: Ah, thanks for the clarification!

**Pull request merged**